docs/StorIOSQLite.md

```diff
@@ -73,6 +73,27 @@ storIOSQLite
   });
 ```
 
+###### Third case: Observe changes of tags to have more fine-grained control over notifications
+
+```java
+storIOSQLite
+  .get()
+  .listOfObjects(Tweet.class)
+  .withQuery(Query.builder()
+    .table("tweets")
+    .observesTags("particular_change_tag") // Subscribe on some specific change
+    .build())
+  .prepare()
+  .asRxObservable();
+```
+
+Also you can handle changes of tags manually
+
+```java
+storIOSQLite
+  .observeChangesOfTag("particular_change_tag");
+```
```

Comment:
Add a comment: `// map/flatMap/filter`?  So it'll be more clear that some transformation of the change is possible